modules/retail/pages/retail-components.adoc

```diff
@@ -1,13 +1,8 @@
 [[retail-components]]
 = Components
 
-{smr} uses a layered architecture:
-
-* The first layer contains the {susemgr} server and one or more build hosts.
-* The second layer contains one or more branch servers to provide local network and boot services.
-* The final layer contains any number of deployed point-of-service terminals.
-
-image::retail_ref_arch.png[scaledwidth=80%]
+{smr} is made up of various components.
+For more on these components work together, see xref:modules/retail/pages/retail-network-arch.adoc[].
```

Comment:
Is this a complete sentence? At least, it's rather dense.  Better inserting "how" after "on"?